scripts/ec2_bootstrap_ubuntu.sh

```diff
@@ -73,26 +65,12 @@ unzip -o awscliv2.zip
 npm install -g aws-cdk
 
 # Setup Docker repository and install Docker components
-apt-get install -y uidmap docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
-install -m 0755 -d /etc/apt/keyrings
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
-chmod a+r /etc/apt/keyrings/docker.asc
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
-  tee /etc/apt/sources.list.d/docker.list > /dev/null
-
-# Add ubuntu user to the docker group
-usermod -aG docker $USERNAME
-
-# Reset the instance
-newgrp docker
-
-# Setup rootless Docker for the user
-su - "$USERNAME" -c "dockerd-rootless-setuptool.sh install"
-
-# Install Python packages
-su - "$USERNAME" -c "pip3 install geojson pytest boto3"
+snap install docker
 
 # Clone primary AWS repo and configure git
-su - "$USERNAME" -c "git clone https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git && cd guidance-for-overhead-imagery-inference-on-aws && git-lfs pull && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules && git submodule update --init --recursive"
-
+git clone https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git
+chmod -R 777 guidance-for-overhead-imagery-inference-on-aws
+cd guidance-for-overhead-imagery-inference-on-aws || exit
```

Comment:
Replace all `guidance-for-overhead-imagery-inference-on-aws` with `guidance-for-processing-overhead-imagery-on-aws`